src/responder/common/cache_req/plugins/cache_req_host_by_name.c

```diff
@@ -59,6 +60,9 @@ cache_req_host_by_name_lookup(TALLOC_CTX *mem_ctx,
     *_result = result;
 
     return EOK;
+#else
+    return ERR_INTERNAL;
```

Comment:
You should never see ERR_INTERNAL in sssd log files. It is kind of `assert false`. But we do not use asserts in sssd.

Debug message will help because you will never see it. Plugin is used only by ssh responder and you cannot see debug message if ssh responder is not built :-)